fastify.js

```diff
@@ -350,7 +350,8 @@ function fastify (options) {
     listeningOrigin: {
       get () {
         const address = this.addresses().slice(-1).pop()
-        /* istanbul ignore if windows: unix socket is not testable on Windows platform */
+        /* ignore if windows: unix socket is not testable on Windows platform */
+        /* c8 ignore next 3 */
```

Comment:
🤢 c8 doesn't support specifying the exception to ignore? It only supports `next` and `next N lines`? That is going to be a real pain to maintain as code evolves.

Reply:
Maybe it got smarter over time, not sure